classes/StockManager.php

```diff
@@ -152,6 +152,7 @@ public function is_woocommerce_loaded() {
     public function load_plugin_textdomain() {
         $locale = is_admin() && function_exists( 'get_user_locale' ) ? get_user_locale() : get_locale();
         $locale = apply_filters( 'plugin_locale', $locale, 'woocommerce-stock-manager' );
+        unload_textdomain( 'woocommerce-stock-manager' );
         load_textdomain( 'woocommerce-stock-manager', WP_LANG_DIR . '/woocommerce-product-stock-alert/woocommerce-product-stock-alert-' . $locale . '.mo' );
```

Comment:
Please change to:

`load_textdomain( 'woocommerce-stock-manager', WP_LANG_DIR . '/plugins/woocommerce-product-stock-alert-' . $locale . '.mo' );`

Thanks.